pkgs/development/tools/build-managers/gradle/default.nix

```diff
@@ -230,7 +230,13 @@ rec {
       } // lib.optionalAttrs (updateAttrPath != null) {
         updateScript = nix-update-script {
           attrPath = updateAttrPath;
-          extraArgs = [ "--url=https://github.com/gradle/gradle" ];
+          extraArgs = [
+            "--url=https://github.com/gradle/gradle"
+            # Gradle’s .0 releases are tagged as `vX.Y.0`, but the actual
+            # release version omits the `.0`, so we’ll wanto to only capture
+            # the version up but not including the the trailing `.0`.
+            "--version-regex=^v(\\d+\\.\\d+(?:\\.[1-9]\\d?)?)(\\.0)?$"
```

Comment:
Right now this’ll cause [nix-update](https://github.com/Mic92/nix-update) to crash, but I’ll open an issue.

Reply:
I'm not sure how to feel about including this flag while it's broken in upstream nix-update. Maybe we should comment it out and/or add it once it's fixed upstream?